Shuttle-back/src/main/java/com/shuttle/ride/dto/BaseRideDTO.java

```diff
@@ -0,0 +1,73 @@
+package com.shuttle.ride.dto;
+
+import java.time.ZonedDateTime;
+import java.util.Collection;
+
+import com.shuttle.location.dto.LocationPairDTO;
+
+// used in notification and user getRides
+// FIX ALL
+public class BaseRideDTO {
```

Comment:
I wouldn't do inheritance in DTOs, just make them all flat classes.

Reply:
This was a debate for violating DRY but you are right, I agree